rpcs3/Emu/CPU/CPUThread.cpp

```diff
@@ -359,7 +359,7 @@ void cpu_thread::operator()()
 		std::this_thread::sleep_for(1ms);
 	}
 
-	if (id_type() == 1 && false)
+	if (false && id_type() == 1)
```

Comment:
Tbh you can probably use a switch or similar for id_type.
It's redundant to check for id_type 2 below anyway, if the id_type is 1

Reply:
Well, actually you are right, but I wasn't correcting any actual logic, especially in CPU area. In this particular area I fixed a lazy evaluatable check. I will look into this.

As for the others, will be done soon.